requirements.txt

```diff
@@ -1,23 +1,25 @@
-click>=8.0.4
+click~=8.1.2
```

Comment:
We changed this and some others to more flexible `>=` following Spencer's PR. I think it was needed for Python 3.6 version. Although IceCube also upgraded, I don't know if we will ever need py3.6

Reply:
I think we are not going to support python 3.6 since IceCube no longer requires it there have been no other requests for it.

Reply:
Okay. We also had this one for py3.6 compatibility;
https://github.com/SNEWS2/SNEWS_Publishing_Tools/blob/05e02890a9223debea4b7745f47a06dae005b5af/snews_pt/snews_format_checker.py#L3-L6

I guess we also don't need that anymore but I also think that it doesn't hurt to keep it.